lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -400,21 +406,44 @@ def cloud(self, value: Optional[CloudSchema]) -> None:
             self.cloud_raw = value.to_dict()  # type: ignore
 
 
+class StaticAccessTokenCredential(TokenCredential):
+    def __init__(self, token: str, expires_on: int) -> None:
```

Comment:
The code can be simplified by chanage to `def __init__(self, token: str) -> None:`, and then parse the expiration time in the method. When it's used, parse expiration date in the token itself.